ops/ccs-ops-misc/load_test/common/db.py

```diff
@@ -52,12 +57,19 @@ def get_hashed_mbis(uri: str, table_sample_pct: Optional[float] = None) -> List:
     else:
         table_sample_text = f"TABLESAMPLE SYSTEM ({table_sample_pct}) "
 
-    bene_query = (
-        f'SELECT "mbi_hash" FROM "beneficiaries" {table_sample_text} WHERE "mbi_hash" IS NOT NULL '
+    # Handle possible hash collisions from duplicate hashes in beneficiaries_history
+    # by only taking MBI hashes that are distinct in both beneficiaries _and_
+    # benficiaries_history
+    mbi_query = (
+        f"SELECT beneficiaries.mbi_hash FROM beneficiaries {table_sample_text} "
+        "    INNER JOIN beneficiaries_history "
+        "        ON beneficiaries.mbi_hash = beneficiaries_history.mbi_hash "
+        "    GROUP BY beneficiaries.mbi_hash "
+        "    HAVING count(beneficiaries_history.mbi_hash) = 1 "
```

Comment:
I'd be curious of the performance of _this_ suggestion. Indeed, the alternative was taken because the original query performed poorly on non-trivially sized data sets–ceteris paribus, the original query takes greater than 7 minutes to return data in production.

Reply:
So @andrew-kramer after watching @mjburling test the proposed query and the existing query (that is slow on large datasets like `prod`) we found that the existing query is somewhat faster by about 2x (with a `LIMIT` of 100,00, like in the code, we saw the existing query return in 6s and the proposed query return in 22s).

In the future we are going to rip this out entirely and stick with a known set of contiguous beneficiary IDs once the data is in a better state. When that happens we should see _significantly_ better performance here, since it will just be a simple `SELECT`.